Dp-202-shop/server/src/db/models/Order.model.ts

```diff
@@ -8,8 +8,6 @@ export interface OrderAttributes {
   total_sum: number;
   status?: string;
   id?: number;
-  buyer_id?: number;
-  salesman_id?: number;
   createdAt?: Date;
   updatedAt?: Date;
```

Comment:
don't delete anything from db without informing. Put it back. It's the fields that are not necessary to enter